website/docs/r/compute_instance_group.html.markdown

```diff
@@ -97,3 +97,11 @@ exported:
 * `self_link` - The URI of the created resource.
 
 * `size` - The number of instances in the group.
+
+## Import
+
+Instance group can be imported using the `zone` and `name`, e.g.
```

Comment:
After talking with @danawillow and Evan, we decided that import method should:
- Always supports the fully specified resource identifier (implemented in this PR).
- For convenience, we can add support for partial specification by `name` only if guessing the other parameters is feasible. The logic for guessing should be in the ImportFunc and not the read method to avoid polluting the read method's logic. If more than one resource match the name (e.g. two instance groups in two different zones), fail with an error message to avoid surprising behavior.
- We will go ahead and submit this PR as is. I will add support for the convenient import by name only in a separate PR.
- We should update the import method for all the other resources to support the fully specified resource. Otherwise, in the case where two resources have the same name in two different zones or regions, the user cannot import the resource.